spec/rails_helper.rb

```diff
@@ -19,10 +19,6 @@
 
 require 'rspec/rails'
 
-# Prevent Test::Unit's AutoRunner from executing during RSpec's rake task on
-# JRuby
-Test::Unit.run = true if defined?(Test::Unit) && Test::Unit.respond_to?(:run=)
-
```

Comment:
This was a hack needed because we were using `test-unit` for Rails 3.2. Definitely unrelated to the ruby 1.9 removal. I'll move the change to the right commit.